### DISCUSSION TITLE
a full / partial COSMOS PDF Documentation ?

### DISCUSSION THREAD

**Discussion by Lion1xx** [CosmosOS/Cosmos#2771] [General]
**Hello !**
 im not using CosmosOS for the first time, idk maybe now for 3yrs, but im struggling to find a full documenation about the VBECanvas/Canvas and Memory management, and some other things in the Cosmos and IL2CPU, anyway if someone has a offline pdf documentation or online, ill be very grateful if you share with me.
thank you for time（＾－＾ )

**Comment by zarlo** [comment]
https://cosmosos.github.io/api/Cosmos.Build.Common.html is the closest thing

**Reply by Lion1xx** [reply]
Thanks man i will check this out ! 😁
